docs/reference/tab-widgets/inference-api/infer-api-reindex.asciidoc

```diff
@@ -155,6 +155,29 @@ might affect the throughput of the reindexing process. If this happens, change
 
 // end::azure-ai-studio[]
 
+// tag::google-vertex-ai[]
```

Comment:
Not part of this PR, but should we link to the reindex API in general in this step and add maybe one sentence why we're using it (basically simulating ingestion)?